rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -224,11 +226,32 @@ static SUPPORTED_SIG_ALGS: WebPkiSupportedAlgorithms = WebPkiSupportedAlgorithms
 /// All defined key exchange groups supported by aws-lc-rs appear in this module.
 ///
 /// [`ALL_KX_GROUPS`] is provided as an array of all of these values.
+/// [`DEFAULT_KX_GROUPS`] is provided as an array of this provider's defaults.
 pub mod kx_group {
     pub use super::kx::{SECP256R1, SECP384R1, X25519};
+    pub use super::pq::{MLKEM768, X25519MLKEM768};
 }
 
-pub use kx::ALL_KX_GROUPS;
+/// A list of the default key exchange groups supported by this provider.
+///
+/// This does not contain MLKEM768; by default MLKEM768 is only offered
+/// in hybrid with X25519.
```

Comment:
re the comment message: 

> This maintains the existing `ALL_KX_GROUPS` and
introduces `DEFAULT_KX_GROUPS`.  These are different
because (for now) `DEFAULT_KX_GROUPS` does not contain
ML-KEM-768.

> That is out of abundance of caution -- in case ML-KEM
does not fare well in its first years of being (partially)
load-bearing for internet traffic.  This seems the most
conservative choice.

Fair enough. There is (at time of writing) activity in TLSWG to adopt and possibly make the MLKEM-only key agreements more 'real' and on track to become an RFC than they currently are, including possibly 'Recommended=Y': if that happens I may raise shuffling the `DEFAULT_KX_GROUPS` around again 😅